.rubocop.yml

```diff
@@ -41,10 +41,8 @@ Lint/UnneededSplatExpansion:
 Lint/IneffectiveAccessModifier:
   Enabled: false
 
-# TODO: enable it in a future because `ruby -w` warns it
 Lint/AmbiguousRegexpLiteral:
-  Enabled: false
+  Enabled: true
 
-# TODO: enable it in a future because `ruby -w` warns it
 Lint/AmbiguousOperator:
-  Enabled: false
+  Enabled: true
```

Comment:
`Lint/Ambiguous*` cops are enabled by default. You don't have to enable them explicitly (unless all `Lint/*` have been disabled explicitly..) IMO, just removing these cops from the config should suffice..

Reply:
Done.